docs/manual/gettingStarted/configuringPlaynite.md

```diff
@@ -0,0 +1,66 @@
+# Configuring Playnite
+
+---------------------
+
+## Accessing Playnite settings
+
+To access Playnite's settings:
+
+1. Click the `Playnite` icon to open the main menu.
+2. Click `Settings…`
+
+![Accessing Playnite Settings](images/gettingStarted_acessSettings.png)
+
+### General configuration
+
+In the **General** section of the settings, you'll find options to configure Playnite's behavior when starting, minimizing, and whether Playnite should start in Fullscreen Mode, among other settings.
+
+![General Configuration](images/gettingStarted_GeneralConfig.png)
+
+### User Interface and appearance configuration
+
+Playnite provides numerous customization options to tailor the user interface and appearance to your preferences. Customize the visibility of elements, change the application font, adjust the placement of panels, modify the height of covers, adjust spacing, and more.
+
+Explore these options in the `Appearance` section to make Playnite your own.
+
+![User Interface and Appearance Configuration](images/gettingStarted_SettingsAppearance.png)
+
+### Importing Playtime from Third-party Libraries and Services
+
+In this settings section, you'll find the "Import playtime of games in library" setting. This setting configures whether playtime provided by third-party services should be imported into Playnite during library updates.
+
+> [!WARNING] 
+> Keep in mind that support by the library plugins responsible for handling the games is required to use this feature. Only library plugins that internally obtain playtime data from their respective service will allow Playnite to import it. This usually requires the user to be logged in and/or configure a key in the specific plugin settings page.
+
+The following modes are available for this setting:
+
+| Mode                          | Description                                                                                                     |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| Always                        | Imports playtime for new imported games added during library updates and existing games in Playnite's database. |
+| Only for newly imported games | Imports playtime only for new imported games added during library updates.                                      |
+| Never                         | Never imports playtime under any circumstance.                                                                  |
+
+> [!NOTE]
+> If you want to keep your playtime always in sync with your game library, use the "Always" setting
+
+### Auto Close Clients
+
+You can configure Playnite to automatically close game clients after you close a game in the **Auto Close Clients** section. This ensures that game clients are only running when needed and not wasting unnecessary resources.
+
+![Auto Close Clients](images/gettingStarted_AutoCloseClients.png)
+
+### Backups support
```

Comment:
Should include link to backup.md

Reply:
https://github.com/JosefNemec/PlayniteDocs/pull/1/commits/ac35f67e5304f61bf9b6cf7c905ae9aadedff0ed